### PR TITLE
Make sombody's day survey 

### PR DESCRIPTION
**Pull request by BamStugendal**
Link to my deployed project:
https://make-somebodys-day.netlify.app/

### CODE REVIEW

File: code/src/Form.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react'
+
+import { Header } from './Header'
+import { FirstQuestion } from './FirstQuestion'
```
Comment (LillanEriksson) [inline code/src/Form.js:4]: Could it be even clearer if you had named each question-component alike? Like question1, question2 on so on?

File: code/src/Form.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react'
+
```
Comment (LillanEriksson) [inline code/src/Form.js:2]: Great that you have a blank line between the import react and the imports of components!

File: code/src/Form.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    const handlerCaptionChange = newCaption => {
+        setCaption(newCaption);
+    }
+
+    const onSubmit = event => {
+        setHideForm(true);
+    }
+
+    console.log(descriptions)
+
```
Comment (LillanEriksson) [inline code/src/Form.js:46]: All console logs should be removed before a pull request :)

File: code/src/Header.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+
+
+export const Header = () => {
+    return (
+        <section className="header"> 
+            <h1>Make somebody's day</h1>
+            <img src="./hands.png" alt="high five" />
+        </section>
+    )
```
Comment (LillanEriksson) [inline code/src/Header.js:11]: The header looks soo nice! But when I viewed the page I was a bit confused over where the questions were, it took me a while to get that I needed to scroll down, I clicked everywhere to se if it was something i should click. Maybe a little notice like "scroll down to answer"? :)

**Review by LillanEriksson — CHANGES_REQUESTED** [review]
Nice and clean code, very readable. You could think about commenting your code more to make it even more readable. The page looks really nice, and the theme you chose for your survey is heartwarming :)
I would only like you to go over your code and remove the console logs.

File: code/src/Summary.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+export const Summary = ( {name, descriptions, gift, cheesyCaption, userName} ) => {
+    
+    return (
+        <section className="summary section">
+            <h2>Hi {name}</h2>
+            <p>You are {descriptions.join(', ')} and if it wasn't for corona I would give you {gift}.</p>
+            <p>{cheesyCaption}</p>
+            <p>Sending love from {userName}</p>
+        </section>
```
Comment (HIPPIEKICK) [inline code/src/Summary.js:11]: Love this summary! 💯

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Such a sweet survey! Requesting changes because the code is inconsistent regarding the usage of semicolons. They should be used everywhere or not at all.

File: code/src/Header.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```
Comment (HIPPIEKICK) [inline code/src/Header.js:1]: Only one left! Approving now 👍

**Pull request closed without merging**